src/components/ClusterManager.js

```diff
@@ -138,6 +167,7 @@ export const NewClusterModal = withModalDrawer({ width: 650 })(class NewClusterM
     this.state = {
       profile: matchingProfile ? matchingProfile.name : 'custom',
       jupyterUserScriptUri: '',
+      selectedLeoImage: leoImages.Default,
```

Comment:
nit: Technically we're storing more state than needed here. Not a big deal since we'll likely end up refactoring in the next PR anyway, but wanted to mention it for onlookers.